lib/pipeline.js

```diff
@@ -4,9 +4,15 @@
 
 const BaseModel = require('./base');
 const parser = require('screwdriver-config-parser');
+const workflowParser = require('screwdriver-workflow-parser');
 const hoek = require('hoek');
+const { PR_JOB_NAME } = require('screwdriver-data-schema').config.regex;
 const PAGINATE_PAGE = 1;
 const PAGINATE_COUNT = 50;
+const PR_NAME_GROUP = {
```

Comment:
can you call this `REGEX_CAPTURING_GROUP` ? Took a while to figure out this maps to `regex` used here https://github.com/screwdriver-cd/data-schema/commit/2e8596e87415aea93cab7085c6b5764368e86e19#diff-77dea1a61b658aff6ad9cd0d2528d2daR26